pkg/gen/nodejs-templates/helm/v2/helm.ts

```diff
@@ -189,11 +193,16 @@ export class Chart extends yaml.CollectionComponentResource {
                 // > is done.
                 const release = shell.quote([releaseName]);
                 const values = path.quotePath(overrides.name);
+                const apiVersionsArgs = cfg.apiVersions
+                    ? cfg.apiVersions
+                        .map((apiVersion) => `--api-versions ${shell.quote([apiVersion])}`)
+                        .reduce((accumulator, currentValue) => accumulator + currentValue)
```

Comment:
It looks like this will produce multiple `--api-versions A --api-versions B`.  But I believe the supported syntax is `--api-versions A,B`, right?  Possibly `.join` is better here?

Reply:
Yes, I tested it with multiple `--api-versions` flags and it worked. `.join` is definitely cleaner though. I'll update that, thanks!

Reply:
Updated. It need to be of the form: `--api-versions={A,B}` with the curly braces and equals sign. I couldn't get it to work with the `shell.quote` because it would add a backslash to escape the `,` which doesn't work. Is the `shell.quote` required here?

Reply:
> Is the shell.quote required here?

Not sure what the format of legal api versions is here - it may be that they can contain characters that need to be quoted?  But if so - you could map the `shell.quote` prior to joining.

Reply:
Good idea, I mapped the `shell.quote` before the join.